sshcommand

```diff
@@ -58,6 +58,12 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=$NAME" "$USERHOME/.ssh/authorized_keys" || true)
```

Comment:
@michaelshobbs Yeah sure :-) I'm unfamiliar with bash testing though. Can I run the test suite on OSX? Or do I have to use a Vagrant box?